openstudiocore/src/openstudio_app/Resources/populate_loads.rb

```diff
@@ -0,0 +1,85 @@
+require 'openstudio'
```

Comment:
@jmarrec why was it necessary to update all the osms?  they all defaulted to these values already right?

Reply:
See comment below: need to hardset the last field.